components/brave_rewards/browser/rewards_notification_service_impl.cc

```diff
@@ -110,6 +110,13 @@ void RewardsNotificationServiceImpl::GetAllNotifications() {
   OnGetAllNotifications(rewards_notifications_list);
 }
 
+void RewardsNotificationServiceImpl::GetAllNotifications(
```

Comment:
you should use `*` instead of `&` when initializing a parameter like this for clarity, but it doesn't seem necessary either way. Why wouldn't we just return a const RewardsNotificationList for `rewards_notifications_`? Also the method itself should be `const`

Reply:
actually why do we need this method at all? We really don't need the _impl.cc class here anyway, but just put `rewards_notifications_` in rewards_notification_service.h